src/utils/ADempiere/constants/actionsMenu.js

```diff
@@ -0,0 +1,63 @@
+// ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
+// Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A.
```

Comment:
Delete it, will be implemented with own window

Reply:
I would apply the similar operation of 

https://github.com/EdwinBetanc0urt/adempiere-vue/blob/develop/src/components/ADempiere/Field/FieldOptions/fieldOptionsList.js